card.io/src/main/AndroidManifest.xml

```diff
@@ -6,6 +6,9 @@
 
     <uses-permission android:name="android.permission.CAMERA"/>
     <uses-permission android:name="android.permission.VIBRATE"/>
+    <!-- We need this for tessdata-->
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
tessdata is an external directory read in by Tesseract. It needs to be included with the app/assets/bundle. With android the only way to do this is have it on the sd card or emulated storage build into most devices. Read/write external in this case does not mean external storage but rather the external emulated storage.